hydrant/adapters/sites/skagit.py

```diff
@@ -46,3 +47,10 @@ def items(self):
             if value:
                 yield attr, value
 
+    def unique_key(self):
+        """Returns a key value to represent this item as unique
+
+        Defined by adapters wishing to sort out duplicates, used in
+        comparison operators.
```

Comment:
Would defining `__eq__()` or `__hash__()` have the same effect?

Reply:
yes, but this feels overreaching.   i don't want to hate myself a year down the road when i compared two Patients and ignored changes as they compared equal...